src/components/NavSection.vue

```diff
@@ -1,10 +1,12 @@
 <template>
   <nav class="header nav">
     <a class="logo" href="/">
-      <img src="../assets/cc-search-logo-white.png">
+      <img src="../assets/cc-search-logo-white.png" alt="logo">
     </a>
     <input class="menu-btn" type="checkbox" id="menu-btn" />
-    <label class="menu_icon" for="menu-btn"><span class="navicon"></span></label>
+    <label class="menu_icon" for="menu-btn">
```

Comment:
This is the hamburger menu icon for mobile screens. It already has a `display: none` on wider screen sizes. Could you follow some of the guidelines listed here instead? https://medium.com/@linlinghao/accessibility-for-hamburger-menu-a37fa9617a89